src/swsssdk/scripts/sonic-db-cli

```diff
@@ -1,11 +1,24 @@
 #!/usr/bin/python
+from __future__ import print_function
 import sys
 import swsssdk
 import redis
 import argparse
 from multiprocessing import Pool
 
-def ping_single_instance(inst_info):
+def ping_unix_path_single_instance(inst_info):
+    inst_hostname = inst_info['hostname']
+    inst_unix_socket_path = inst_info['unix_socket_path']
+    r = redis.Redis(host=inst_hostname, unix_socket_path=inst_unix_socket_path)
+    rsp = False
+    msg = 'Could not connect to Redis at {}:{}: Connection refused'.format(inst_hostname, inst_unix_socket_path)
+    try:
+        rsp = r.ping()
+    except redis.exceptions.ConnectionError as e:
+        pass
+    return 'PONG' if rsp else msg
+
+def ping_tcp_single_instance(inst_info):
```

Comment:
>ping_tcp_single_instance [](start = 4, length = 24)

Most code duplicated in ping_tcp_single_instance and ping_unix_path_single_instance. Not a blocking issue, and with you could improve them.